client/components/deposits-overview/next-deposit.tsx

```diff
@@ -64,19 +147,22 @@ const NextDepositDetails: React.FC< NextDepositProps > = ( {
 					<FlexItem className={ `${ tableClass }__cell` }>
 						<Loadable
 							isLoading={ isLoading }
-							value={ strings.tableHeaders.nextDepositDate }
+							value={ __(
+								'Estimated dispatch date',
+								'woocommerce-payments'
+							) }
 						/>
 					</FlexItem>
 					<FlexItem className={ `${ tableClass }__cell` }>
 						<Loadable
 							isLoading={ isLoading }
-							value={ strings.tableHeaders.status }
+							value={ __( 'Status', 'woocommerce-payments' ) }
 						/>
 					</FlexItem>
 					<FlexItem className={ `${ tableClass }__cell` }>
 						<Loadable
 							isLoading={ isLoading }
-							value={ strings.tableHeaders.amount }
+							value={ __( 'Amount', 'woocommerce-payments' ) }
```

Comment:
I also went on a tear moving strings inline. This might have been a bit ambitious – I could be convinced to back out 😁